### PR TITLE
Display playlists on artist page

### LINKED ISSUE #318: FEATURE: Show artist playlists on his page
In itunes and the player on iOS popular artists have a list of playlists such as Esentials, Next Steps.

The playlists are available through the API:
`playlists = await music.api.artist(id, { include: 'playlists' });`

In order to minimize the requests to the API it can be combined into the albums request:
`await music.api.library.artist(id, { include: ['albums', 'playlists'] });`

### PR DESCRIPTION
**Pull request by SpartakusMd**
Fixes #318.
This feature will display the playlists for artists that exists on Apple Music.

image

### CODE REVIEW

File: src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx [hunk 1]
```diff
@@ -132,13 +151,28 @@ class ArtistPage extends React.Component {
...
         {albums && (
-          <div className={classes.albumsGrid}>
-            {albums.map(album => (
-              <AlbumItem key={album.id} album={album} size={120} />
-            ))}
+          <div>
+            <h3>{translate.albums}</h3>
+
+            <div className={classes.albumsGrid}>
+              {albums.map(album => (
```
[the remark this answers is not in the document]
Comment (SpartakusMd) [reply src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx]: Is there some misconfiguration ? In the IDE I get the inverse notice:
```ESLint: Replace `<AlbumItem·key={album.id}·album={album}·size={120}·/>` with `⏎················<AlbumItem·key={album.id}·album={album}·size={120}·/>⏎··············` (prettier/prettier)```

[the remark this answers is not in the document]
Comment (BrychanOdlum) [reply src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx]: Yup, there completely is. We’ve had this a few times when mapping a component. Never gotten around to fixing it 😕

File: src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx [hunk 2]
```diff
@@ -132,13 +151,28 @@ class ArtistPage extends React.Component {
...
+                <AlbumItem key={album.id} album={album} size={120} />
+              ))}
+            </div>
+          </div>
+        )}
+
+        {playlists && (
+          <div>
```
Comment (BrychanOdlum) [inline src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx]: Away from my laptop this evening so can’t test right now. But does this need to be a div or could this be made a fragment (<>) instead? Don’t want to add anything unnecessary to the DOM

(Same for albums)

Comment (SpartakusMd) [reply src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx]: Cool. I didn't knew about fragments. Thanks.

Comment (BrychanOdlum) [reply src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx]: No problem :) They’re a cool lil’ feature!

Thanks for your Pr though! Will test tomorrow before merging :)

**Comment by BrychanOdlum** [comment]
Looks good :)

**Pull request merged**